src/Cli.Tests/EndToEndTests.cs

```diff
@@ -241,7 +242,7 @@ public void TestAddEntityWithoutIEnumerable()
     public Task TestConfigGeneratedAfterAddingEntityWithoutIEnumerables()
     {
         string[] initArgs = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--database-type", "mssql", "--connection-string", "localhost:5000",
-            "--set-session-context", "true" };
+            "--set-session-context", "true", "--runtime.base-route", "/base-route" };
```

Comment:
The `"--runtime.base-route` is out of place here since this test isn't validating that behavior.